puller.Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Build the inference-puller binary
+FROM golang:1.13.0 as builder
```

Comment:
We should move to 1.14 asap

Reply:
No worries there, should I do that in this PR? Or is that for another time